tipz.zsh

```diff
@@ -61,7 +61,7 @@ function _tipz_process {
     # If the search exited successfully,
     # output the tip to the user
     if [[ $? -eq 0 ]]; then
-      echo "\033[1;34mTipz:\033[0;m \033[0;34m$result ${(@)cmd:$i}\033[0;m"
+      echo "\033[1;34m${ZSH_PLUGINS_TIPZ_TEXT:-Tipz:}\033[0;m \033[0;34m$result ${(@)cmd:$i}\033[0;m"
```

Comment:
Could you please remove the `ZSH_PLUGINS_` prefix from the variable? And quote the fallback text 'Tipz:' as well, just to be safe.